docs/_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-cayman
```

Comment:
You can remove this as we don't use jekyll, we use asciidoctor instead. The files are built as plain html files before being pushed to `gh-pages` (see https://github.com/killbill/killbill-docs/tree/gh-pages/latest).

Reply:
I meant remove the whole file.